### ISSUE TITLE
GdkX11not working despite pyObject present

### ISSUE THREAD

**Issue by Vanderscycle** [ibus/ibus#2370]
Hey, thank you for maintaining this lib.

TL;DR: can't get ibus-setup working on endavoursOS using Xmonad as my wm. 

Please fill in the following items if you don't know the root cause.

Which distribution and version?: 
EndavourOS

Which desktop environment and version?: 
Xmonad

Which session type?: 
x11

Which application and version?: 
kitty

IBus version?: 
1.5.25

Issue description:
```
Traceback (most recent call last):
  File "/usr/share/ibus/setup/main.py", line 35, in <module>
    gi_require_version('GdkX11', '3.0')
  File "/home/henri/miniconda3/lib/python3.9/site-packages/gi/__init__.py", line 126, in require_version
    raise ValueError('Namespace %s not available' % namespace)
ValueError: Namespace GdkX11 not available
```
my py env (using conda)
```pygobject                 3.42.0           py39ha6f447c_1    conda-forge```

Steps to reproduce:
1. 

Can you reproduce your problem when you restart ibus-daemon? (yes / no): 
(Run `ibus exit` and `ibus-daemon --xim &`) yes, same Traceback error

Do you see any errors when you run ibus-daemon with the verbose option?:
(Run `ibus-daemon --xim --verbose &` and look at the output when you encounter your problem.)
```
(ibus-ui-gtk3:42809): IBUS-WARNING **: 13:17:02.158: panel.vala:265: If you launch KDE5 on xterm, export XDG_CURRENT_DESKTOP=KDE before launch KDE5.

(ibus-ui-gtk3:42809): IBUS-WARNING **: 13:17:02.180: ibus_bus_call_sync: org.freedesktop.DBus.Properties.Get: GDBus.Error:org.freedesktop.DBus.Error.Failed: No global engine.
```

Can you reproduce your problem with a new user account instead of the current your account? (yes / no): 
I don't think it will make a difference, but I can try on xfce4 later

Is there a requirement.txt that I can use?

Thank you for the help.

**Comment by fujiwarat** [comment]
I think you don't install GdkX11 package.
Please ask EndavourOS maintainers the package name.

In Fedora:
```
% rpm -qf /usr/lib64/girepository-1.0/GdkX11-3.0.typelib 
gtk3-3.24.31-2.fc35.x86_64
```

**Comment by Vanderscycle** [comment]
@fujiwarat 
Thank you for the quick answer, Fcitx allowed me to have chinese input on endavourOs. Ibus worked for me when I was on Manjaro.

content of my /etc/environment file
```
GTK_IM_MODULE=fcitx
QT_IM_MODULE=fcitx
XMODIFIERS=@im=fcitx
```

**Comment by fujiwarat** [comment]
Again, please ask your distro.
